package.json

```diff
@@ -32,7 +32,7 @@
       "eslint --fix --max-warnings 0",
       "git add"
     ],
-    "!(**/*dist/**/*).{js,jsx}": [
```

Comment:
Why are we removing `dist` completely?

Reply:
@moshthepitt dist seems to hold compiled js which is compiled to lower js version.  should this go through linting ?

Reply:
I agree with that.  My questions is that it seems your change removes the exclusion of the dist directories

Reply:
@moshthepitt  sorry, I read it wrong. I did exclusion of eslintrc.js and dist folder on the package.json and it felt it would too much so I chose to add the exclusions to .eslintignore